tests/clusterintegrationtest/_include.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+#
+# Copyright 2025 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+function start_tidb() {
+  cd ../../ || exit 1
+  echo "building tidb-server..."
+  make
+  echo "build successfully"
+
+  cd - || exit 1
+
+  echo "Starting TiUP Playground in the background..."
+  tiup playground nightly --db=1 --kv=1 --tiflash=1 --db.binpath=../../bin/tidb-server &
```

Comment:
1. how to use it in release branches in feature?
2. how to use it in CI jobs in pull requests of `tiflash` repo?

Reply:
How to test against a possible command like `tiflash=pr/xxxx`?